lib/Swagger/src/TemplateClassification/Storage/Models/TemplateTypeProvider.php

```diff
@@ -34,222 +42,260 @@
 namespace Swagger\Client\TemplateClassification\Storage\Models;
 
 use \ArrayAccess;
+
 /**
  * TemplateTypeProvider Class Doc Comment
  *
- * @category    Class
- * @description 
+ * @category    Class */
+/**
  * @package     Swagger\Client
  * @author      http://github.com/swagger-api/swagger-codegen
- * @license     http://www.apache.org/licenses/LICENSE-2.0 Apache Licene v2
+ * @license     http://www.apache.org/licenses/LICENSE-2.0 Apache License v2
  * @link        https://github.com/swagger-api/swagger-codegen
  */
 class TemplateTypeProvider implements ArrayAccess
 {
     /**
-      * Array of property to type mappings. Used for (de)serialization 
+      * The original name of the model.
+      * @var string
+      */
+    protected static $swaggerModelName = 'TemplateTypeProvider';
+
+    /**
+      * Array of property to type mappings. Used for (de)serialization
       * @var string[]
       */
-    static $swaggerTypes = array(
+    protected static $swaggerTypes = [
         'provider' => 'string',
         'origin' => 'string',
         'types' => 'string[]'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "types" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)